lib/checks/lists/only-listitems.json

```diff
@@ -5,7 +5,10 @@
 		"impact": "serious",
 		"messages": {
 			"pass": "List element only has direct children that are allowed inside <li> elements",
-			"fail": "List element has direct children that are not allowed inside <li> elements"
+			"fail": {
+				"default": "List element has direct children that are not allowed inside <li> elements",
+				"roleNotValid": "List element has direct children with a role that is not allowed"
```

Comment:
I think it's worth mentioning which role it is. Would make this message clearer.